### PR TITLE
Add patient start end date

### PR DESCRIPTION
**Pull request by poojaRal**

### CODE REVIEW

File: controller/handler/create-patient.js [hunk 1]
```diff
@@ -39,7 +39,8 @@ function createPatient (request, reply) {
     })
     // Create the new Patient
     .then(() => {
-        return patient.create({pin}, {transaction});
+        return patient.create({pin, dateStarted: request.payload.patStartDate,
```
Comment (ChristianMurphy) [inline controller/handler/create-patient.js]: style nit :art: 
could you do a new line for each object key?
``` js
{
   pin: ...,
   dateStarted: ...,
   dateCompleted: ...
}
```

File: controller/router.js [hunk 2]
```diff
@@ -83,7 +83,9 @@ module.exports = [
             validate: {
                 payload: {
                     stageId: Joi.number().integer(),
-                    trialId: Joi.number().integer()
+                    trialId: Joi.number().integer(),
+                    patStartDate: Joi.date().min(moment().format('YYYY-MM-DD')),
+                    patEndDate: Joi.date().min(moment().format('YYYY-MM-DD'))
```
Comment (ChristianMurphy) [inline controller/router.js]: naming these `dateStarted` and `dateCompleted` would be clearer on how this maps to db.

**Comment by ChristianMurphy** [comment]
two small :art: changes, other than that :+1:

File: controller/router.js (hunk 2 above)
Comment (poojaRal) [inline controller/router.js]: These names are the same as per DB.

**Comment by ChristianMurphy** [comment]
:shipit:

**Pull request merged**